src/index.ts

```diff
@@ -981,7 +993,7 @@ class Piscina extends EventEmitterAsyncResource {
     return Piscina;
   }
 
-  static move (val : Transferable | TransferListItem | ArrayBufferView) {
+  static move (val : Transferable | TransferListItem | ArrayBufferView | ArrayBuffer | MessagePort) {
```

Comment:
These two should already have been captured through `TransferListItem`, I think?

Reply:
I thought they would too but they weren't in local testing. Not sure why exactly